front/app/containers/Admin/settings/general/index.tsx

```diff
@@ -58,20 +96,22 @@ class SettingsGeneralTab extends PureComponent<
     super(props);
     this.state = {
       attributesDiff: {},
-      tenant: null,
+      appConfiguration: null,
       loading: false,
       errors: {},
       hasUrlError: false,
       saved: false,
+      settingsUpdatedSuccessFully: false,
+      settingsSavingError: false,
     };
   }
 
   componentDidMount() {
-    const currentTenant$ = currentAppConfigurationStream().observable;
+    const appConfiguration$ = currentAppConfigurationStream().observable;
 
     this.subscriptions = [
-      currentTenant$.subscribe((currentTenant) => {
-        this.setState({ tenant: currentTenant.data });
+      appConfiguration$.subscribe((appConfiguration) => {
```

Comment:
This is ok, but not the preferred way to go about fetching data. Since you know that perfectly, I'm guessing you don't have time for a better solution, it's just a small missed opportunity of unifying practices